factory/cluster.go

```diff
@@ -71,14 +71,14 @@ func DestroyCluster(g GinkgoTInterface) (string, error) {
 		return "", shared.ReturnLogError("error getting config: %w", err)
 	}
 
-	varDir, err = filepath.Abs(shared.BasePath() + fmt.Sprintf("/distros-test-framework/config/%s.tfvars", cfg.Product))
+	tfDir, err := filepath.Abs(shared.BasePath() + fmt.Sprintf("/distros-test-framework/modules/%s", cfg.Product))
 	if err != nil {
-		return "", shared.ReturnLogError("invalid product: %s\n", cfg.Product)
+		return "", err
```

Comment:
why taking out the logs ?

Reply:
Will add it back, need to check how it got overwritten